packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -331,6 +362,7 @@ linter:
     RegExp allowedIdentifiersPattern = new RegExp(r'_?([A-Z]|_+)\b');
     RegExp constructorTearOffsPattern = new RegExp('.+#.+// analyzer doesn\'t like constructor tear-offs');
     RegExp ignorePattern = new RegExp(r'// analyzer says "([^"]+)"');
+    RegExp conflictingNamesPattern = new RegExp('^The imported libraries \'([^\']+)\' and \'([^\']+)\' cannot have the same name \'([^\']+)\'\$');
```

Comment:
We don't want to exclude the error, we just want to catch it and remove the temporary filename from the message.